libr/debug/p/debug_native.c

```diff
@@ -1398,8 +1421,9 @@ static int r_debug_native_drx (RDebug *dbg, int n, ut64 addr, int sz, int rwx, i
  * we only handle the case for hardware breakpoints here. otherwise,
  * we let the caller handle the work.
  */
-static int r_debug_native_bp (RBreakpoint *bp, RBreakpointItem *b, bool set) {
+static int r_debug_native_bp (void *bp_, RBreakpointItem *b, bool set) {
```

Comment:
Why was that parameter changed?

Reply:
Removed  warning with  RBreakpointCallback type aka "typedef int (*RBreakpointCallback)(void *bp, RBreakpointItem *b, bool set);"